code/game/machinery/computer/buildandrepair.dm

```diff
@@ -552,116 +544,3 @@
 	if(I.use_tool(src, user, 50, volume = I.tool_volume) && !state)
 		to_chat(user, "<span class='notice'>You deconstruct [src].</span>")
 		deconstruct(TRUE)
-
-
-
-/obj/structure/computerframe/HONKputer
-	name = "Bananium Computer-frame"
-	icon = 'icons/obj/machines/HONKputer.dmi'
-	base_mineral = /obj/item/stack/sheet/mineral/bananium
```

Comment:
The `base_mineral` variable can be removed entirely, since it's only actually used here.